iGCS/Utils/GCSSettings.h

```diff
@@ -0,0 +1,25 @@
+//
+//  GCSSettings.h
```

Comment:
@cnatoli do you think we should append "Model" to all models? We (mostly me) have been a bit inconstant so far.